src/zepben/auth/authenticator.py

```diff
@@ -43,6 +49,12 @@ class ZepbenAuthenticator(object):
     issuer_domain: str
     """ The domain of the token issuer. """
 
+    auth_method: AuthMethod
+    """ The authentication method used by the server """
+
+    verify_certificate: False
```

Comment:
this needs to be `verify_certificate: bool = False`

Reply:
Right you are